src/components/ConfirmDialog.tsx

```diff
@@ -27,6 +27,21 @@ export const ConfirmDialog = ({
 }: ConfirmDialogProps) => {
   const { t } = useTranslation();
 
+  useEffect(() => {
+    const keyDownHandler = (event: KeyboardEvent) => {
+      if (event.key === 'Enter') {
+        event.preventDefault();
+        onAccept();
+      }
+    };
+
+    document.addEventListener('keydown', keyDownHandler);
+
+    return () => {
+      document.removeEventListener('keydown', keyDownHandler);
+    };
+  });
```

Comment:
bare note to later: viktig å huske dependency array for useEffect, for å unngå at den kjører for hver render.
```suggestion
  }, []);
```